gamemodes/terrortown/gamemode/shared/sh_sprint.lua

```diff
@@ -65,15 +65,14 @@ if SERVER then
 		ply.isSprinting = bool
 	end)
 else
-	-- The helptext can't be changed once the convar was created, so we go with english, since it is probably the most common lang
-	local doubletap_sprint_anykey = CreateClientConVar("ttt2_doubletap_sprint_anykey", 0, true, false, LANG.GetTranslationFromLanguage("doubletap_sprint_anykey", "english"), 0, 1)
-	local disable_doubletap_sprint = CreateClientConVar("ttt2_disable_doubletap_sprint", "0", true, false, LANG.GetTranslationFromLanguage("disable_doubletap_sprint", "english"), 0, 1)
+	local enable_doubletap_sprint = CreateConVar("ttt2_enable_doubletap_sprint", "1", {FCVAR_ARCHIVE})
```

Comment:
This isn't shared. It's still only created on the client. `CreateClientConVar` is only a proxy for the `CreateConVar` function, see ref: https://wiki.facepunch.com/gmod/Global.CreateClientConVar

At least I prefer the latter, because it clearly states which arguments are set.